components/profile/sidebarcard/src/Avatar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```

Comment:
I would like that's acceptable right now, but it would be a good idea to think that this component would potentially be an independent Sui Avatar component. :)

Reply:
Iup we've talked today with our UX+UI guys about the same thing. In the future as we are going to work closely with UI-UX to be sui compliant we will made a definition of this component and export it as an atom component.